src/py42/sdk/queries/fileevents/file_event_query.py

```diff
@@ -34,14 +34,20 @@ def __str__(self):
         groups_string = u",".join(
             str(group_item) for group_item in self._filter_group_list
         )
-        json = u'{{"groupClause":"{0}", "groups":[{1}], "pgNum":{2}, "pgSize":{3}, "srtDir":"{4}", "srtKey":"{5}"}}'.format(
-            self._group_clause,
-            groups_string,
-            self.page_number,
-            self.page_size,
-            self.sort_direction,
-            self.sort_key,
-        )
+
+        items = []
+        items.append(u'"groupClause":"{}"'.format(self._group_clause))
+        items.append(u'"groups":[{}]'.format(groups_string))
+        items.append(u'"srtDir":"{}"'.format(self.sort_direction))
+        items.append(u'"srtKey":"{}"'.format(self.sort_key))
+        if self.pgToken is not None:
+            items.append(u'"pgToken":"{}"'.format(self.pgToken))
+        else:
+            items.append(u'"pgNum":{}'.format(self.page_number))
+            items.append(u'"pgSize":{}'.format(self.page_size))
```

Comment:
`pgSize` can still be used in combination with `pgToken`.

Reply:
Added.